src/app/package.json

```diff
@@ -21,13 +21,17 @@
     "@testing-library/react": "^13.2.0",
     "@testing-library/user-event": "^14.2.0",
     "@types/jest": "^28.1.8",
+    "@types/leaflet-draw": "^1.0.6",
     "@types/node": "^18.7.13",
+    "@types/polylabel": "^1.0.5",
```

Comment:
Normally I'd suggest that types go in the `devDependecies`, but for this project it doesn't really matter since we're not actually building/running a server in production (it's just a bundle).

And to anticpate the question about why some types are in the regular dependecies: I'm not really sure. I think that's just the way CRA sets them up.